inspirehep/modules/theme/templates/inspirehep_theme/header.html

```diff
@@ -40,327 +41,106 @@
-                    <a class="text-link-decoration small-screens-a"><i class="fa fa-line-chart"></i> Links and stats</a>
-                  </span>
-                  <i id="seventh-arrow" class="fa fa-chevron-right arrow-position"></i>
-                </div>
-              </div>
-              <div id="links-and-stats-dropdown" class="popover custom-popover">
-                <h3 class="popover-title" id="links-and-stats-header"><i class="fa fa-line-chart"></i> Links and stats
-                  <div class="row">
-                    <div class="col-md-6"><hr class="custom-horizontal-line"></div>
-                    <div class="col-md-6"></div>
-                  </div>
-                </h3>
-                <div class="popover-content">
-                  <div class="row">
-                    <div class="col-md-3">
-                      <h5 class="text-left"><ins><a class="cursor-pointer popover-link external-link" href="http://inspirehep.net/info/hep/stats/topcites/index">Top cites</a></ins></h5>
-                      <p class="text-left font-p">All the publications with the most citations.</p>
-                    </div>
-                    <div class="col-md-3">
-                      <h5 class="text-left"><ins><a class="cursor-pointer popover-link">Other databases</a></ins></h5>
-                      <p class="text-left font-p">Partners, directories and resources.</p>
-                    </div>
-                    <div class="col-md-6"></div>
-                  </div>
-                  <div class="row"><div class="col-md-12"><p></p></div></div>
-                  <div class="row"><div class="col-md-12"><p></p></div></div>
-                  <div class="row">
-                    <div class="col-md-3">
-                      <h5 class="text-left"><ins><a class="cursor-pointer popover-link external-link" href="http://inspirehep.net/search?of=hcs&action_search=Search">HEP Citesummary</a></ins></h5>
-                      <p class="text-left font-p">Statistics concerning HEP related information.</p>
-                    </div>
-                    <div class="col-md-3">
-                      <h5 class="text-left"><ins><a class="cursor-pointer popover-link">External resources</a></ins></h5>
-                      <p class="text-left font-p">Non-HEP related resources.</p>
-                    </div>
-                    <div class="col-md-6"></div>
-                  </div>
-                </div>
-              </div>
-            </li>
-          </ul>
-        </li>
-        <!-- End Nice dropdown  -->
-
-          <li class="move-down">
+        <div class="row" id="mobile-menu">
+          <div class="col-sm-3 col-xs-4 col-xxs-6">
+            <div class="mobile-menu-square"></div>
+            <a href="/literature" class="mobile-menu-thumbnail">
+              <img src="{{ url_for('static', filename='images/mobile-menu-icon-literature.svg') }}">
+              <p id="literature-collection">Literature</p>
```

Comment:
This `id` names could potentially be used in other pages, would be better to have a prefix unique of the mobile menu.